src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -185,12 +190,16 @@ public void setDeadlineTaskFilter(Predicate<? super DeadlineTask> predicate) {
         filteredDeadlineTasks.setPredicate(predicate);
     }
 
+    @Override
+    public void setIsFinishedDeadlineFilter() {
+        setDeadlineTaskFilter(new DeadlineTaskFinishedPredicate());
+    }
```

Comment:
@nightism Looks like you can just do `setDeadlineTaskFilter(new DeadlineTaskFinishedPredicate())` in `HideFinishedDeadlineCommand`, no?